### ISSUE TITLE
detect_system_plugins requires bundler or inspec to be installed

### ISSUE THREAD

**Issue by zenspider** [inspec/inspec#4431]
```
inspec_gemspec = Gem::Specification.find_by_name("inspec", "=#{Inspec::VERSION}")
```

means that you _must_ use bundler or have the gem installed for that line to work, and that happens on immediate invocation (including just plain `--version`).

There needs to be a different way to figure out the dependencies for the gem than this... or it should be done in a way that will elegantly recover.

**Comment by clintoncwolfe** [comment]
Can you describe circumstances at runtime under which inspec would not be installed?

**Comment by zenspider** [comment]
In a development environment, eg: `minitest test/unit/some/test/file.rb`.

Our code should not be dependent upon how it is installed (or not), nor how it is being run. I worked all our bundler specific code out of this system to make it more flexible in a variety of scenarios and I'd like to keep it that way.

**Comment by zenspider** [comment]
Fixed by #4513 which will go out today.
